cmd/kueuectl/app/list/list_workload.go

```diff
@@ -63,8 +68,13 @@ type WorkloadOptions struct {
 	ClusterQueueFilter string
 	LocalQueueFilter   string
 	StatusesFilter     sets.Set[int]
+	forGVK             schema.GroupVersionKind
+	forName            string
+
+	UserSpecifiedForObject string
 
 	ClientSet clientset.Interface
+	Result    *resource.Result
```

Comment:
Result of what? This needs a better name.

Reply:
Renamed